src/app/series/directives/series-feed.component.css

```diff
@@ -2,27 +2,39 @@ h4 {
   font-size: 18px;
   line-height: 22px;
   font-weight: 400;
-  margin: 6px 0;
+  margin: 20px 0 6px 0;
   color: #343434;
+  border-bottom: 1px black solid;
+}
+
+ul {
+  padding: 0 5px;
 }
 
 li {
   display: flex;
   justify-content: space-between;
   margin: 5px 0;
+  border-bottom: 1px #ddd solid;
+}
+
+li:last-child {
+  border-bottom: none;
 }
 
-.public {
-  color: #000000;
+li h5 {
+  width: calc(3 * 100%/5);
```

Comment:
can get the same effects by using `li h5 { flex: 3 }` and `li p { flex: 1}` and skipping width, right?